runbotics/runbotics-desktop/src/core/websocket/bpmn/runtime-subscriptions.service.ts

```diff
@@ -264,13 +265,15 @@ export class RuntimeSubscriptionsService {
         });
     }
 
-    private sanitizeVariable(variable: any): string {
-        const variableToCheck: string = JSON.stringify(variable);
-        if (variableToCheck && variableToCheck.length > 100_000)
-            return JSON.stringify(
-                { message: 'Exceeded max length', partialResponse: variableToCheck.slice(0, 100_000)}
-            );
-        return variableToCheck;
+    private sanitizeObject(objectValue: any): any {
```

Comment:
replace any with "unknown" where possible 
also 
I think we could use some comment explaining the general use, params and return value + optionally example simple input, output

Reply:
Local tests with examples: https://all41.atlassian.net/browse/RPA-1247